doc/tutorials/plugins.md

```diff
@@ -356,15 +356,17 @@ and write each key as its own line in the file. Since we don't care about the na
 the value of `keyString` for each `Key` as a new line in the file. That's it. Now, each time the mounted `KeySet` is modified, `elektraPluginSet` will
 be called and the mounted file will be updated.
 
-#### `ELEKTRA_SET_ERROR`
+#### `ELEKTRA_SET_<CONCRETE>_ERROR`
```

Comment:
better word than `<CONCRETE>`? Which are available?

Reply:
I am also not quite sure and found <CONCRETE> the best but what about:
`ELEKTRA_SET_..._ERROR`, `ELEKTRA_SET_<TYPE>_ERROR`,  `ELEKTRA_SET_<CONCRETE_TYPE>_ERROR` ...?

Reply:
Yes, maybe "concrete type". But you should define what a concrete or abstract type is.